### PR TITLE
[RNMobile] remove override support inserter for columns block

### PR DESCRIPTION
**Pull request by jbinda**
## Description

🚧  **NOTE:** Merge only after [Allow support inserter in Columns block PR](https://github.com/WordPress/gutenberg/pull/20502) will be merged. 🚧

This PR removes `support inserter` overrides made in `index.natie.js` inside Column Block.

Please refer to:
[Related gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2221)
[Allow support inserter in Columns block PR](https://github.com/WordPress/gutenberg/pull/20502)
[Column Block PR](https://github.com/WordPress/gutenberg/pull/19013)
[design note](https://github.com/WordPress/gutenberg/pull/19013#issuecomment-596734085)

## How has this been tested?

Checking all Column Block PR testing requirements ([see Column PR description](https://github.com/WordPress/gutenberg/pull/19013#issue-350696448))

List of changes:
- remove `index.native.js` from column with overrides `support: inserter` setting

## Types of changes

Refactor  - Change overrides of `support: inserter` after `support inserter: false` from `index.js` of column block prop will be delete.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/index.native.js
Comment (jbinda) [inline packages/block-library/src/index.native.js]: remember to revert before merge !

File: packages/block-library/src/column/index.js
Comment (youknowriad) [inline packages/block-library/src/column/index.js:24]: I have a similar PR, I believe we need a notice in the UI somewhere when we reach the limit of max 6 columns. It becomes more of a guideline than a requirement.

Comment (jbinda) [reply packages/block-library/src/column/index.js:24]: I merged your PR to this :)

On mobile side we temporary override `support: inserter` to true in `index.native.js` and will remove it after your PR gets merged.

On mobile Columns works slight different because we are wrapping them depending on screen size. It do not visualise the layout in 100% but in the other hand we don't need to worry about the max limit.

I will post you a GIF tomorrow to show what I mean

Comment (jbinda) [reply packages/block-library/src/column/index.js:24]: Below GIFs should visualise what I mean.

**Vertical position**
column vertical

**Horizontal position**
column vert

---------
**Few side notes**

Currently our guidelines according to looks like that: [see this comment](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1430#issuecomment-541180270). However we also have hardcoded limitation on block container max-width set to **580px**. The conclusion is that at this moment we should never have more that 2 Columns in the row on mobile. The limitation notice in Columns on mobile will have more sense if we can go above that width limit ( also above **768px**). Then the Columns logic will try to render all Columns in one row. Still we can adjust to add some min-width limit from which we start to wrap in the same manner again. Maybe someday in the future we extend this max-width - don't know exact plan on this but on iPad it feels quite awkward because of lot of empty space on sides - specially in Horizontal layout. I know it's UX to increase readability but not sure if it wasn't also some workaround because of implementation. Please see below screen to see how it looks on iPad.

Apart of above if we have UI notice on web we also should have it on mobile to notice user that layout may be different on web. I'am fully agree with that. @iamthomasbishop do you have any thoughts how this notice should looks like ?

image

**Comment by jbinda** [comment]
because  [Allow support inserter in Columns block PR](https://github.com/WordPress/gutenberg/pull/20502) is merged I have sync the PR and we can remove `index.native.js` to override `support: inserter` flag

@pinarol do you think we can merge this or wait too see if mentioned PR wont be reverted according to [latest comment](https://github.com/WordPress/gutenberg/pull/20502#issuecomment-624540035)

**Review by pinarol — APPROVED** [review]
LGTM

**Pull request merged**
